src/connx.c

```diff
@@ -642,6 +642,9 @@ int connx_Graph_run(connx_Graph* graph, uint32_t input_count, connx_Tensor** inp
             if (node->outputs[j] == 0)
                 continue;
 
+            if (node->output_names == NULL)
+                continue;
+
```

Comment:
You need to surround these lines using #if DEBUG and #endif /* DEBUG */ because node->output_names field appears only in DEBUG mode.

Reply:
But this entire block is already in `#if DEBUG` condition.
See [L639](https://github.com/semihlab/connx/pull/46/files/29bb6dc283f37435d1608050ebe168855c6f4623#diff-a74466fbdf139868210a16ade07a1df08fb6b779ce8e811aea389e2da524b4e3R639)